db/seeds.rb

```diff
@@ -46,13 +46,11 @@
 karlo.confirm
 karlo.update!(accepted_privacy_policy_at: Time.current, accepted_terms_at: Time.current)
 
-# TODO: Hard-code list of tracks before merging.
-track_slugs = %w[fsharp]
+track_slugs = %w[05ab1e ada arm64-assembly ballerina bash c ceylon cfml clojure clojurescript coffeescript common-lisp coq cpp crystal csharp d dart delphi elixir elm emacs-lisp erlang factor forth fortran fsharp gleam gnu-apl go groovy haskell haxe idris io j java javascript julia kotlin lfe lua mips nim nix objective-c ocaml perl5 pharo-smalltalk php plsql pony powershell prolog purescript python r racket raku reasonml ruby rust scala scheme shen sml solidity swift system-verilog tcl typescript vbnet vimscript x86-64-assembly zig]
 track_slugs.each do |track_slug|
   puts "Adding Track: #{track_slug}"
 
-  # TODO: Change to exercism before merging
-  repo_url = "https://github.com/erikschierboom/#{track_slug}"
+  repo_url = "https://github.com/exercism/#{track_slug}"
```

Comment:
Ah - I was going to leave this unchanged until we merge, so that we can test this PR before Friday.

Reply:
Okay. Want me to revert?